__tests__/app.test.js

```diff
@@ -42,3 +50,26 @@ describe("3. GET /api/topics", () => {
       });
   });
 });
+
+describe("4. GET /api/articles/:article_id", () => {
+  test("status 200: responds with an article object containing the following properties", () => {
+    const article_id = 1;
+    return request(app)
+      .get(`/api/articles/${article_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body).toEqual(
```

Comment:
I'd like you to specifically check for the right articleID to come back in this test (aka 1)

Reply:
Added one more test with a specific article!